### PR TITLE
new post: endorsing the new Scala CoC

### PR DESCRIPTION
**Pull request by larsrh**
Closes typelevel/general#51.

### CODE REVIEW

File: posts/2016-12-17-scala-coc.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Endorsing the new Scala Code of Conduct"
+
+meta:
+  nav: blog
+  author: typelevel
+---
+
+A couple of days ago, the new [Scala Code of Conduct](https://contributors.scala-lang.org/t/please-read-scala-code-of-conduct/28) has been published.
```
Comment (tpolecat) [inline posts/2016-12-17-scala-coc.md]: s/has been/was/

File: posts/2016-12-17-scala-coc.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+It applies to all official Scala channels, including mailing lists, Gitter channels and GitHub repositories.
+We would like to take this opportunity to endorse this new Code of Conduct.
+From our perspective, it does a good job of listing encouraged behaviour, instead of just banning harassment: It reflects the goal of _actively_ creating a welcoming community.
+Also, we consider it to be a decent substitute of our own [Code of Conduct](/conduct.html).
+That means that Typelevel project maintainers are free to switch to the Scala Code of Conduct.
+At this point, there is no need to do so, but we would like to encourage Typelevel project maintainers to consider switching at their own discretion.
```
Comment (tpolecat) [inline posts/2016-12-17-scala-coc.md]: "At this point" hints that at some point there *will* be a need to do so, which has not been discussed and I think should not be brought up here. I suggest removing the last sentence.

Comment (larsrh) [inline posts/2016-12-17-scala-coc.md]: I was trying to make the point that maintainers don't have to switch. Can you suggest alternative wording for that?

File: posts/2016-12-17-scala-coc.md (hunk 1 above)
Comment (larsrh) [inline posts/2016-12-17-scala-coc.md]: *shakes fist at tenses*

File: posts/2016-12-17-scala-coc.md (hunk 2 above)
Comment (tpolecat) [inline posts/2016-12-17-scala-coc.md]: Maybe add to the next-to-last sentence?

*This means that Typelevel project maintainers are free to switch to the Scala Code of Conduct if they wish.*

Comment (larsrh) [inline posts/2016-12-17-scala-coc.md]: Done.

**Pull request merged**